src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -798,6 +798,7 @@ trait BronzeTransforms extends SparkSessionWrapper {
             TransformFunctions.stringTsToUnixMillis('timestamp)
           } else col("Timestamp")
 
+          logger.log(Level.INFO, s"DEBUG: SparkEventsBronze Case Sensitivity set to: ${spark.conf.get("spark.sql.caseSensitive")}")
```

Comment:
Do we want to leave this log? Ok if so, just wanted to make sure you did it on purpose. If you're going to leave it, we probably need to add the orgID in context to the message so we know for which orgID the conf is being set.